Code/GraphMol/ForceFieldHelpers/catch_tests.cpp

```diff
@@ -0,0 +1,56 @@
+
+//
+//  Copyright (C) 2024 Greg Landrum and other RDKit contributors
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include <cmath>
+#include <RDGeneral/test.h>
+#include <catch2/catch_all.hpp>
+
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/DistGeomHelpers/Embedder.h>
+#include <ForceField/MMFF/Params.h>
+#include <ForceField/MMFF/BondStretch.h>
+
+#include "FFConvenience.h"
+
+using namespace RDKit;
+
+TEST_CASE("Test empty force field") {
+  auto mol = "CCCC"_smiles;
+  REQUIRE(mol);
+  REQUIRE_FALSE(DGeomHelpers::EmbedMolecule(*mol));
```

Comment:
Same comment about constructing a molecule that has a conformer already.
This will also allow you to simplify both the headers you include here and the `CMakeLists.txt`